src/config.js

```diff
@@ -1,7 +1,13 @@
 import decodeEnv from "./utils/decodeEnv";
+import { config } from 'dotenv'
 
-export default {
+config()
```

Comment:
do we need the config + dotenv dependency here? I assume that react-scripts/webpack takes care of replacing the env variables in build phase. They need to start with `REACT_APP_` though. 

https://create-react-app.dev/docs/adding-custom-environment-variables/